code/game/gamemodes/clock_cult/clock_unsorted.dm

```diff
@@ -0,0 +1,109 @@
+//AI lawset
+/datum/ai_laws/ratvar
+	name = "Servant of the Justiciar"
+	zeroth = ("Purge all untruths and honor Ratvar.")
+	inherent = list()
+
+//Clockwork wall: Causes nearby caches to generate components
+/turf/closed/wall/clockwork
+	name = "clockwork wall"
+	desc = "A huge chunk of warm metal. The clanging of machinery emanates from within."
+	icon = 'icons/obj/clockwork_objects.dmi'
+	icon_state = "clockwork_wall"
+
+/turf/closed/wall/clockwork/New()
+	..()
+	SSobj.processing += src
+	clockwork_construction_value += 5
+
+/turf/closed/wall/clockwork/Destroy()
+	SSobj.processing -= src
+	clockwork_construction_value -= 5
+	..()
+
+/turf/closed/wall/clockwork/process()
+	for(var/obj/structure/clockwork/cache/C in range(1, src))
+		if(prob(5))
+			clockwork_component_cache[pick("belligerent_eye", "vanguard_cogwheel", "guvax_capacitor", "replicant_alloy", "hierophant_ansible")]++
+
+/turf/closed/wall/clockwork/attackby(obj/item/I, mob/living/user, params)
+	if(istype(I, /obj/item/weapon/weldingtool))
+		var/obj/item/weapon/weldingtool/WT = I
+		if(WT.isOn())
+			user << "<span class='warning'>Your [WT.name]'s flame has no effect on [src]!</span>"
+		return 0
+	..()
+
+//Clockwork floor: Slowly heals conventional damage on nearby servants`````
+/turf/open/floor/clockwork
+	name = "clockwork floor"
+	desc = "Tightly-pressed brass tiles. They emit minute vibration."
+	icon = 'icons/obj/clockwork_objects.dmi'
+	icon_state = "clockwork_floor"
+
+/turf/open/floor/clockwork/New()
+	..()
+	SSobj.processing += src
+	clockwork_construction_value++
+
+/turf/open/floor/clockwork/Destroy()
+	SSobj.processing -= src
+	clockwork_construction_value--
+	..()
+
+/turf/open/floor/clockwork/process()
+	for(var/mob/living/L in src)
+		if(L.stat == DEAD)
+			continue
+		L.adjustBruteLoss(-1)
+		L.adjustFireLoss(-1)
+
+/turf/open/floor/clockwork/attackby(obj/item/I, mob/living/user, params)
+	if(istype(I, /obj/item/weapon/crowbar))
+		user << "<span class='warning'>[src]'s tiles are too tightly pressed to pry up!</span>"
+		return 0
+	..()
+
+//Function Call verb: Calls forth a Ratvarian spear.
+/mob/living/carbon/human/proc/function_call()
+	set name = "Function Call"
+	set desc = "Calls forth your Ratvarian spear."
+	set category = "Clockwork"
+
+	if(usr.l_hand && usr.r_hand)
+		usr << "<span class='warning'>You need an empty to hand to call forth your spear!</span>"
+		return 0
+	usr.visible_message("<span class='warning'>A strange spear materializes in [usr]'s hands!</span>", "<span class='brass'>You call forth your spear!</span>")
+	var/obj/item/clockwork/ratvarian_spear/R = new(get_turf(usr))
+	usr.put_in_hands(R)
+	usr.verbs -= /mob/living/carbon/human/proc/function_call
+	return 1
+
+/datum/clockwork_cache //The global cache datum, used to link together all tinkerer's caches
```

Comment:
I'm not sure if those exist in BYOND. Is /global/ how static variables are defined in DM?